### ISSUE TITLE
Alta/baja de users  e insert de users en  POST /producers

### ISSUE THREAD

**Issue by abc-recursos** [abcrecursos/abc-recursos-be#3]
Si bien el diseño actual de la aplicacion no prevee registracion de donantes y solicitantes que trabajen en entidades de salud, sí prevee que las Pymes y otras organizaciones productoras u solicitantes se registren en la aplicación. Está pendiente todavia implementar en el frontend el registro. En la API supongo entonces que hay que generar el servicio para registrar a estos usuarios. Necesitamos entonces un servicio para insertar usuarios (POST /users) y servicios para consultarlos (para otras funciones de backend). 
El registro de usuario deberia incorporar los datos de contacto, los datos de direccion, y las coordenadas del proceso de georeferenciacion.
Otro issue relacionado es el POST /producers, que falta que incluya el id de usuario.

**Comment by edvm** [comment]
@abc-recursos al registrarse ponen su password ? se van a poder loguear? o solo se dan de alta?

**Comment by abc-recursos** [comment]
@edvm Sí. Iriamos por lo estandar: (usuario), email y contraseña, etc.

**Comment by AlvaDamian** [comment]
Qué seguridad se le va a dar al sistema?.

Hace poco estuve leyende sobre SRP (Secure Remote Password) pero no le he implementado. Creo que podría ser algo bueno en cuestiones de autenticación.

Les dejo unos links:

[SRP en Wikipedia](https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol)
[Librería cliente y servidor de mozilla implementando SRP](https://github.com/mozilla/node-srp)

Tal vez alguien ya lo haya hecho a esto y podría implementarlo fácilmente.
